admin/tabs/general/roles/details/members/RoleMembers.ts

```diff
@@ -109,12 +113,12 @@ const count = hoistCmp.factory<CountProps>(({count, icon}) =>
 );
 
 const bbar = hoistCmp.factory<RoleMembersModel>(({model}) => {
-    const {directCounts, moduleConfig} = model;
-    if (!moduleConfig?.directoryGroupsSupported && directCounts.DIRECTORY_GROUP) {
+    const {effectiveCounts, moduleConfig} = model;
+    if (!moduleConfig?.directoryGroupsSupported && effectiveCounts.DIRECTORY_GROUP) {
         return warningBanner({
             message: 'Directory Groups disabled. Will ignore.'
         });
-    } else if (!moduleConfig?.userAssignmentSupported && directCounts.USER) {
+    } else if (!moduleConfig?.userAssignmentSupported && effectiveCounts.USER) {
```

Comment:
This actually isn't correct.  A Role can (and should) have effective users when `userAssignmentSupported = false`.  That flag is perhaps a little misleading - it refers to the **direct** assignment of users.  Effective members will **not** be ignored, so I would remove these warnings here altogether.